pkgs/top-level/all-packages.nix

```diff
@@ -18713,10 +18719,6 @@ in
 
   jflex = callPackage ../development/libraries/java/jflex { };
 
-  junit = callPackage ../development/libraries/java/junit { antBuild = releaseTools.antBuild; };
-
-  junixsocket = callPackage ../development/libraries/java/junixsocket { };
```

Comment:
Those need alias entries.

Reply:
`junixsocket` depends on `junit`. `junit` has been marked as broken since 2015-09-08, i.e. 6 years ago. Nobody has submitted a patch for `junit` during the past 6 years. Any downstream who currently unbreaks `junit` will be fine vendoring it; those downstreams basically have been vendoring `junit` for the past 6 years.

This lack of need of aliases is now better explained in the message of the commit that removes them.